electionleaflets/apps/api/feeds.py

```diff
@@ -28,42 +28,6 @@ def item_description(self, item):
             d = "{0} – {1}".format(d, item.images.all()[0].image.url)
         return d
 
-    def item_enclosure_url(self, item):
-        if item.images.all():
-            return item.images.all()[0].image.url
-
-    def item_enclosure_length(self, item):
-        if item.images.all():
-            try:
-                return item.images.all()[0].image.size
-            except:
-                return None
-
-    def item_enclosure_mime_type(self, item):
-        if item.images.all():
-            im_type, _ = mimetypes.guess_type(item.images.all()[0].image.url)
-            return im_type
-
-
-# class PartyFeed(Feed):
-#     title = "electionleaflets.org latest party leaflets"
-#     description = "The most recently uploaded party leaflets"
-#
-#     def get_object(self, request, party_slug):
-#         obj = get_object_or_404(Party, slug=party_slug)
-#         self.link = "/parties/%s/" % obj.slug
-#         return obj
-#
-#     def items(self,obj):
-#         return Leaflet.objects.filter(publisher_party=obj).order_by('-id')[:10]
-#
-#     def item_title(self, item):
-#         return item.title
-#
-#     def item_description(self, item):
-#         return item.description
-
-
 class ConstituencyFeed(Feed):
```

Comment:
We seem to be dropping most of the constituencies app in this PR. Do we definitely want to keep this?